installer-tests/Vagrantfile

```diff
@@ -60,6 +65,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
   end
 
+  # Use qemu driver for libvirt. See README.md for rationale.
+  config.vm.provider :libvirt do |libvirt|
+    libvirt.driver = 'qemu'
```

Comment:
Here would be a good place to adjust memory/CPU offers, since we seem to only do that for virtualbox above.

Feel free to steal the block from https://github.com/zarvox/vagrant-spk/blob/master/vagrant-spk#L60-98 , which computes cpus/ram to offer once, and then applies it to each provider.